samples/transform_feedback_instanced.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - transform_feedback_instanced</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="utility.js"></script>
+    <script src="third-party/gl-matrix-min.js"></script>
+    <script src="third-party/noise3D.js"></script>
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - transform_feedback_instanced</div>
+    <p id="description">Demonstrates how to combine transform feedback and instanced drawing. Agents are wandering randomly.</p>
+
+    <!-- WebGL 2 shaders -->
+    <script id="vs-emit" type="x-shader/x-vertex">
+        #version 300 es
+        #define OFFSET_LOCATION 0
+        #define ROTATION_LOCATION 1
+
+        #define M_2PI 6.28318530718
+
+        #define MAP_HALF_LENGTH 1.01
+        #define WANDER_CIRCLE_R 0.01
+        #define WANDER_CIRCLE_OFFSET 0.04
+        #define MOVE_DELTA 0.001

```

Comment:
Since the simulation is a bit more involved, it would be nice to briefly described the constants. Also, if you have a link to a resource describing how the direction is updated, that could be helpful too.